setup.py

```diff
@@ -139,12 +142,25 @@ def run(self):
 
 
 if __name__ == '__main__':
-    version = detect_model_server_version()
+    # Get nightly version if nightly in name
+    name = 'torchserve'
+
+    # Clever code to figure out if setup.py was trigger by ts_scripts/push_nightly.sh
+    NAME_ARG = "--override-name"
+    if NAME_ARG in sys.argv:
+        idx = sys.argv.index(NAME_ARG)
+        name = sys.argv.pop(idx + 1)
+        sys.argv.pop(idx)
```

Comment:
Please use [ArgumentParser](https://docs.python.org/3/library/argparse.html)
```suggestion
    from argparse import ArgumentParser
    parser = ArgumentParser()
    parser.add_argument("--override-name", default='torchserve')
    args = parser.parse_args()
    name = args.override_name
    # Get nightly version if nightly in name
```

Reply:
+1